File: resize_logo.js
const sharp = require('sharp');
const path = require('path');

async function processImages() {
    const inputPath = path.join(__dirname, 'public', 'logo_extracted.png');
    const iconPath = path.join(__dirname, 'src', 'app', 'icon.png');
    const logoPath = path.join(__dirname, 'public', 'logo.png');

    try {
        // 144x144 for icon.png
        await sharp(inputPath)
            .resize(144, 144, { fit: 'contain' })
            .png()
            .toFile(iconPath);
        console.log('Successfully generated src/app/icon.png');

        // copy to public/logo.png
        await sharp(inputPath)
            .png()
            .toFile(logoPath);
        console.log('Successfully generated public/logo.png');
    } catch (err) {
        console.error(err);
    }
}

processImages();

File: resize_logo.py
from PIL import Image

# Read the extracted image
img = Image.open('public/logo_extracted.png')

# Resize it to 144x144, keeping transparency
img = img.resize((144, 144), Image.LANCZOS)

# Save as app/icon.png
img.save('src/app/icon.png')
print("Successfully generated src/app/icon.png")

# Also save as public/logo.png for the navbar to use
img_large = Image.open('public/logo_extracted.png')
img_large.save('public/logo.png')
print("Successfully saved public/logo.png")
